src/resolvers/http-and-file.ts

```diff
@@ -13,11 +15,24 @@ export const httpAndFileResolver = createHttpAndFileResolver();
 export function createHttpAndFileResolver(opts?: IHttpAndFileResolverOptions): Resolver {
   const resolveHttp = createResolveHttp({ ...DEFAULT_REQUEST_OPTIONS, ...opts });
 
+  // Note: this should be better fixed at @stoplight/json-ref-readers level
+  const patchedResolved = (ref: URI): Promise<unknown> => {
+    return new Promise((resolve, reject) => {
+      const path = ref.valueOf();
```

Comment:
given `c:/toto/file.txt`, `pathname()` returns `/toto/file.txt`. This is was causing the main resolving havoc.